tracee-ebpf/tracee/tracee.go

```diff
@@ -741,6 +741,27 @@ func (t *Tracee) populateBPFMaps() error {
 	bpfConfigMap.Update(uint32(configFollowFilter), boolToUInt32(t.config.Filter.Follow))
 	bpfConfigMap.Update(uint32(configDebugNet), boolToUInt32(t.config.Debug))
 
+	// Initialize pid_to_cont_id_map if tracing containers
+	c := Containers{}
+	err := c.Populate()
+	if err != nil {
+		return err
+	}
+	bpfPidToContIdMap, _ := t.bpfModule.GetMap("pid_to_cont_id_map")
+	for _, contId := range c.GetContainers() {
+		for _, pidstr := range c.GetPids(contId) {
+			if t.config.Debug {
+				fmt.Println("Running container =", contId, "pid =", pidstr)
+			}
+			var pid uint32
+			_, err = fmt.Sscanf(pidstr, "%d", &pid)
```

Comment:
leftover: I was checking and removed thinking there was no way it couldn't be a %d in that file (because its a procfs file).. but we never know, I'm putting it back.